Update to latest package_version

milljm opened the pull request:
update documentation to match.

Closes #13338

Commit:
Update to latest package_version

update documentation to match.

Closes #13338

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/329333/) on dca7019 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/13339/site)

This comment will be updated on new commits.

Commit:
Update toolchain instructions

With the update to GCC, the toolchain path has changed. Or perhaps it was always
incorrect.

moosebuild commented:
Job [Code Coverage](https://civet.inl.gov/job/329350/) on dca7019 wanted to post the following:

View code coverage site [here](https://mooseframework.inl.gov/docs/PRs/13339/coverage)

moosebuild commented:
Job [Code Coverage](https://civet.inl.gov/job/330827/) on 1ade48b wanted to post the following:

View code coverage site [here](https://mooseframework.inl.gov/docs/PRs/13339/coverage)

moosebuild commented:
All [jobs](https://civet.inl.gov/event/41772/) on 0fc7241 : canceled by @milljm

Failed miserably at selecting which targets to test. Try again.

moosebuild commented:
Job [Test timings](https://civet.inl.gov/job/331847/) on ce593cb wanted to post the following:

View timings [here](https://mooseframework.inl.gov/docs/PRs/13339/timing)

This comment will be updated on new commits.

moosebuild commented:
Job [Code Coverage](https://civet.inl.gov/job/331823/) on ce593cb wanted to post the following:

View code coverage site [here](https://mooseframework.inl.gov/docs/PRs/13339/coverage)

permcody commented:
Merging! Note that failed targets are optional and have existing failures.
